### PR TITLE
Automatically change to "Follow Wario" when creating new room

### PR DESCRIPTION
**Pull request by Goldensunboy**
Fix save corruption #263

### CODE REVIEW

File: WL4EditorWindow.cpp [hunk 1]
```diff
@@ -4,6 +4,7 @@
 #include "ROMUtils.h"
 #include "ui_WL4EditorWindow.h"
 #include "Themes.h"
+#include "LevelComponents/Room.h"
```
Comment (shinespeciall) [inline WL4EditorWindow.cpp:7]: this line not needed.

**Review by shinespeciall — CHANGES_REQUESTED** [review]

File: WL4EditorWindow.cpp (hunk 1 above)
Comment (Goldensunboy) [reply WL4EditorWindow.cpp:7]: It is needed for the __CameraControlType enum used on line 1955

Comment (Goldensunboy) [reply WL4EditorWindow.cpp]: Okay it is not needed

**Review by shinespeciall — APPROVED** [review]
LET'S.

**Pull request merged**